### DISCUSSION TITLE
Specific Actor Reload Macros

### DISCUSSION THREAD

**Discussion by MrVauxs** [JDCalvert/FVTT-PF2e-Ranged-Combat#65] [General]
Half a question, half a suggestion if not implemented.

Is it possible to write a Reload macro that affects only one actor?

The use case for this in particular is less to having one macro per actor, but have the ability to not have the token selected when reloading, and instead have the actor defined another way (like automatically using Automated Animations).

**Comment by MrVauxs** [comment]
Preferably it should be possible to just supply the TokenPF2e in question.
ex. ```let token = args[1].sourceToken; console.log(token); game.pf2eRangedCombat.reload(token);```
![image](https://user-images.githubusercontent.com/32039708/175115774-001d7da1-d41d-48f3-8f45-779d320afab1.png)

**Comment by JDCalvert** [comment]
Hi there, so at the moment the way the module finds the actor/token to act on is actually the selected token _or_, if you have no token selected, your assigned character's token, as long as you only have exactly one token.
